storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2169,6 +2169,8 @@ static bool innodb_init_param(void) {
 
   srv_force_recovery = (ulint)innobase_force_recovery;
 
+  dblwr::g_mode = dblwr::Mode::OFF;
```

Comment:
I would rather set it in modules that do not need to be ON.
like stats, prepare, backup for now.

Reason is, we might want to allow dblwr to be ON for prepares.

Reply:
This is common to all functionalities, so avoid it here

Reply:
I am also disabling it inside storage/innobase/log/log0log.cc. 
`
} else if (str_starts_with(creator_name, LOG_HEADER_CREATOR_PXB)) {
      dblwr::g_mode = dblwr::Mode::OFF;
      dblwr::set();

      if (srv_read_only_mode) {
        ib::error() << "Cannot restore from xtrabackup, InnoDB running in "
                       "read-only mode!";
        return DB_ERROR;
      }
      `

Reply:
@rahulmalik87 why do it here? if we do at the top of prepare, isn't it sufficient?

Reply:
yeah, It got duplicated. I will remove it from log0log.cc

Reply:
Thanks. Mark it as resolved once you have addressed it